cognite/neat/_rules/models/dms/_validation.py

```diff
@@ -150,6 +154,64 @@ def validate(self) -> IssueList:
         issue_list.extend(self._validate_referenced_container_limits(dms_schema.views, view_properties_by_id))
         return issue_list
 
+    def _duplicated_resources(self) -> IssueList:
+        issue_list = IssueList()
+
+        properties_sheet = self._read_info_by_spreadsheet.get("Properties")
+        views_sheet = self._read_info_by_spreadsheet.get("Views")
+        containers_sheet = self._read_info_by_spreadsheet.get("Containers")
+
+        visited = set()
+        for row_no, property_ in enumerate(self._properties):
+            if property_._identifier() in visited:
+                issue_list.append(
+                    ResourceDuplicatedError(
+                        property_.view_property,
+                        "property",
+                        (
+                            "the Properties sheet at row "
+                            f"{properties_sheet.adjusted_row_number(row_no) if properties_sheet else row_no + 1}"
+                            " if data model is read from a spreadsheet."
+                        ),
+                    )
+                )
+            visited.add(property_._identifier())
```

Comment:
Suggest doing something like this. Then you handle more than 2 duplicates as one error. 
```suggestion
visited = defaultdict(list)
for row_no, property_ in enumerate(self._properties):
    visited[property_._identifier()].append(row_no)

for identifier, row_numbers in visited.items():
    if len(row_numbers) == 1:
        continue
    issue_list.append(
        ResourceDuplicatedError(
            identifier,
            "property",
            (
                "the Properties sheet at row "
                f"{humanize_collection([properties_sheet.adjusted_row_number(row_no) if proerties_sheet for row_no in row_numbers])}"
                " if data model is read from a spreadsheet."
            ),
        )
    )
```